RELEASE.md

```diff
@@ -0,0 +1,18 @@
+# RELEASE 1.0.0
+
+## New Features
+
+- Improve the Graph feature to include interactive response. Users can now click on a node to inspect node info and parameter
```

Comment:
- Improve the Graph feature to include interactive response. Users can now click on a node to inspect its details such as parameters, input, output, dimensions, etc. They can also zoom in/out and move around if the graph is too large, restore to its original size and download it as PNG.